modules/apps/product-navigation/product-navigation-applications-menu-web/src/main/resources/META-INF/resources/css/ApplicationsMenu.scss

```diff
@@ -2,13 +2,10 @@
 
 .applications-menu-modal {
 	.modal-full {
```

Comment:
If not used anywhere else we might want to replace `modal-full` with `modal-dialog`

Reply:
Sure if the styles are preserved I think we can replace it :)

Reply:
Updating
![image](https://user-images.githubusercontent.com/46778114/100210581-72acb900-2f0b-11eb-95c8-145781d53e0a.png)